src/components/segments/MeetupsContainer.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import _ from 'lodash';
+import moment from 'moment';
+
+import MeetupDate from './MeetupDate';
+
+class MeetupsContainer extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      loading: true,
+      meetups: []
+    }
+  }
+
+
+  componentWillMount() {
+    const url = "https://codeforamerica.org/api/organizations/Code-for-Denver/upcoming_events?per_page=50";
```

Comment:
Fyi, if you just request `https://codeforamerica.org/api/organizations/Code-for-Denver`, it has a key for `current_events` that has the next two events. Might be easier to work with.